app/_locales/en/messages.json

```diff
@@ -3052,6 +3058,12 @@
   "testFaucet": {
     "message": "Test Faucet"
   },
+  "theme": {
+    "message": "Theme"
+  },
+  "themeDescription": {
+    "message": "Switch to the metamask dark mode."
```

Comment:
Can someone suggest a better copy for this `themeDescription`?

Reply:
I think we can just do "Switch to $1 mode", then substitute in the "dark" or "default'

Reply:
updated